### PR TITLE
Move features that were not backported to 8.9 to the 8.10 section of CHANGES.md.

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** documentation fix

### CODE REVIEW

**Comment by silene** [comment]
Thanks for the patch. That was an item from my TODO list, which I can now remove.

I don't remember which project it was, but they had a nice system to avoid this mess. They created a `changes` directory, in which every pull request was adding its own file describing the changes. Then, at release time, the manager would concatenate all these files into `CHANGES.md` and empty the directory. This had two advantages. First, the changelog was always synchronized with the release. Second, pull requests never caused merge conflicts on the `CHANGES.md` file.

**Review by silene — APPROVED** [review]

**Comment by Zimmi48** [comment]
Indeed, that's also my plan for future releases (see #7058, #7075 and #8151).

**Comment by Zimmi48** [comment]
@silene Don't wait for a merge conflict before merging this PR :laughing:

**Pull request merged**
